### PR TITLE
Use SHLIB_EXT

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

**Comment by jschueller** [comment]
ping @scopatz

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,16 +13,13 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 200
```
Comment (scopatz) [inline recipe/meta.yaml]: Not that it matters, but why such a large increase in the build number?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -66,10 +63,14 @@ test:
...
     - jinja2
+  source_files:
+    - pyne
+    - src
+    - tests
 
 about:
   home: http://pyne.io/
-  license: BSD 3-clause
+  license: BSD 3-Clause
```
Comment (scopatz) [inline recipe/meta.yaml]: Both versions are correct

**Comment by scopatz** [comment]
Frankly, I am shocked that this worked. Thanks for the effort!

File: recipe/meta.yaml (hunk 1 above)
Comment (jschueller) [inline recipe/meta.yaml]: I think we're supposed to start at 200 for recipes that include the blas variant feature

Comment (scopatz) [inline recipe/meta.yaml]: Where is that mentioned?

Comment (jschueller) [inline recipe/meta.yaml]: I don't know, maybe ask @jakirkham

Comment (scopatz) [inline recipe/meta.yaml]: That seems incorrect to me. And if it is correct, it needs documentation.  @conda-forge/core, any thoughts?

Comment (jakirkham) [inline recipe/meta.yaml]: See this [hackpad]( https://conda-forge.hackpad.com/BLAS-Numpy-Friends-86De62hoNdT ) for details.

Comment (scopatz) [inline recipe/meta.yaml]: I see. Would you mind migrating that to the docs? Or making an issue to migrate it please?  Thanks!

**Comment by ocefpaf** [comment]
Better yey. Check if that is still needed.
A lot has changed in conda.

On Jan 17, 2017 7:17 PM, "Anthony Scopatz" <notifications@github.com> wrote:

*@scopatz* commented on this pull request.
------------------------------

In recipe/meta.yaml <https://github.com/conda-forge/pyne-feedstock/pull/8>:

> @@ -13,16 +13,13 @@ source:
   sha256: {{ sha256 }}

 build:
-  number: 0
+  number: 200

I see. Would you mind migrating that to the docs? Or making an issue to
migrate it please? Thanks!

—
You are receiving this because you are on a team that was mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/conda-forge/pyne-feedstock/pull/8>, or mute the thread

**Pull request merged**